.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,6 +11,7 @@ A clear and concise description of what the bug is.
 
 **Amplify CLI Version**
 You can use `amplify -v` to check the amplify cli version on your system
+In case of regression also mention the prior working Amplify version
```

Comment:
nit: make casing of Amplify/amplify CLI/cli consistent

I prefer: "In case of a regression, specify the last known working Amplify CLI version" 
@ammarkarachi what do you think?